### PR TITLE
All encounter cards for War of the Outer Gods

### PR DESCRIPTION
**Pull request by grabben2**

### CODE REVIEW

File: pack/side/wog_encounter.json [hunk 1]
```diff
@@ -0,0 +1,1035 @@
...
+        "position": 49,
+        "quantity": 5,
+        "text": "Swarming 2. Warring.\n<b>Spawn</b> - Burning Pit.\n<b>Forced</b> - When Trylogog attacks: It deals either its damage or its horror, instead of both.",
+        "traits": "Creature. Insect.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "86050",
+        "encounter_code": "swarm_of_assimilation",
+        "encounter_position": 8,
```
Comment (drawntotheflame) [inline pack/side/wog_encounter.json:926]: position 9 please.

File: pack/side/wog_encounter.json [hunk 2]
```diff
@@ -0,0 +1,1035 @@
...
+        "position": 44,
+        "quantity": 3,
+        "text": "Aloof. Retaliate. Warring.\n<b>Spawn</b> - Shrine of Magh'an Ark'at.\n<b>Forced</b> - At the end of the round: Place 1 resource on Bringer of Paradise, as a mutation. Then, if there are 3 or more mutations on it, remove 2 of them and place 2 doom on the green agenda.",
+        "traits": "Humanoid. Mutated.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "86045",
+        "encounter_code": "children_of_paradise",
+        "encounter_position": 7,
```
Comment (drawntotheflame) [inline pack/side/wog_encounter.json:831]: position 8 please.

File: pack/side/wog_encounter.json [hunk 3]
```diff
@@ -0,0 +1,1035 @@
...
+        "position": 38,
+        "quantity": 3,
+        "text": "Warring.\n<b>Spawn</b> - Athenaeum of the Empty Sky.\n<b>Forced</b> - After Ethereal Entity engages you: You must decide (choose one) -\n- Place 1 doom on the blue agenda.\n- Place 1 of your clues on your location.\n- Take 2 horror.",
+        "traits": "Monster.",
+        "type_code": "enemy"
+    },
+    {
+        "code": "86039",
+        "encounter_code": "death_of_stars",
+        "encounter_position": 7,
```
Comment (drawntotheflame) [inline pack/side/wog_encounter.json:717]: Position 8 please (changing the positions to make them reflect the quantities of the additional enemy).

File: pack/side/wog_encounter.json [hunk 4]
```diff
@@ -0,0 +1,1035 @@
...
+        "illustrator": "Robert Laskey",
+        "name": "Blade of Ark'at",
+        "pack_code": "wog",
+        "position": 54,
+        "quantity": 1,
+        "skill_combat": 2,
+        "skill_wild": 1,
+        "slot": "Hand",
+        "subname": "Shellblade Tribute",
+        "text": "[free] Exhaust Blade of Ark'at and spend 1 resource: Place 1 resource on Blade of Ark'at.\n[action]: <b>Fight</b>. You get +1 [combat] for each resource on Blade of Ark'at. If you succeed, remove up to 3 resources from Blade of Ark'at to deal that much additional damage to the attacked enemy.",
```
Comment (drawntotheflame) [inline pack/side/wog_encounter.json:1011]: For the first ability, please add in italics '(from the token pool)' after Blade of Ark'at: 
Place 1 resource on Blade of Ark'at <i>(from the token pool)</i>.

Comment (drawntotheflame) [reply pack/side/wog_encounter.json:1011]: For Fight, please move the full stop inside the bold tag: <b>Fight.</b>

**Review by drawntotheflame — COMMENTED** [review]
Great job, @grabben2

**Comment by grabben2** [comment]
Fixed after @drawntotheflame reviewed it.

**Comment by drawntotheflame** [comment]
@Kamalisk this can go live now - thanks.

**Pull request merged**